### ISSUE TITLE
Kilka par token + refresh token

### ISSUE THREAD

**Issue by maniootek** [allegro/allegro-api#8627]
Jestem początkującym w temacie REST API.
Chciałem się zorientować, czy da się pracować na kilku parach token + refresh token?
Jedna aplikacja miała by swoje dane a druga swoje.

**Comment by arekpietrzyk** [comment]
@maniootek większy sens techniczny, nieporównywalnie większe bezpieczeństwo i stabilność działania - nie mówiąc już o optymalizacji zużycia limitu zapytań do API - uzyskasz tworząc osobne aplikacje w portalu developerskim dla każdej niezależnej aplikacji po swojej stronie.

Skoro masz już potrzebę budowy niezależnych aplikacji, jaki jest sens korzystania z tej samej pary kluczy dostępowych do API?

**Comment by maniootek** [comment]
czy da się z treści tokena / refresh tokena odczytać jego datę ważności?

**Comment by PrzemyslawLukanowski** [comment]
Tak, możesz [rozkodować token](https://jwt.io/), zobaczysz m.in. informację właśnie o dacie ważności.
